docs/references.rst

```diff
@@ -43,6 +43,10 @@ Glossary
       of measuring a given reconstructed energy as a function of the true photon
       energy. See :ref:`irf-edisp`
 
+    FoV
+        Stands for "field of view", and indicates the solid angle on the sky that is
```

Comment:
Strictly speaking it's mostly not given by the solid angle, but by the radius of the (circular) FoV, maybe precise...

Reply:
Even in your formulation you are implying that the FoV is circular (i.e. a solid angle) ;) no?

I don't know, sometimes one says "a FoV of [...] sr", and sometimes gives just the radius. Either use is fine I think. I'll try to clarify

Reply:
I guess its up to us to define the term, whether we mean solid angle or radius. In the context of IACTs I have mostly heard "FoV of 5 deg" or similar...